src/librbd/internal.cc

```diff
@@ -2356,7 +2356,35 @@ bool compare_by_name(const child_info_t& c1, const child_info_t& c2)
     }
   }
 
+  int list_watchers(ImageCtx *ictx,
+		    std::list<librbd::image_watcher_t> &watchers)
+  {
+    int r;
+    std::string header_oid;
+    std::list<obj_watch_t> obj_watchers;
+
+    if (ictx->old_format) {
+      header_oid = ictx->name + RBD_SUFFIX;
```

Comment:
Nit: use ```util::old_header_name```

Reply:
Thanks! Was unaware of these util functions. Updated.